### ISSUE TITLE
robyn_write() model performance metrics show NRMSE for the validation set not the test set. Is it normal?

### ISSUE THREAD

**Issue by SeanRichterWalsh** [facebookexperimental/Robyn#823]
## Describe issue
In the model exported when using robyn_write(), the performance metric for NRMSE is on the validation set not the test set. But adjusted R2 is reported for the test set. Why is the NRMSE of the test set not reported? 

Here are the modelling metrics from an example one-pager:

![Screenshot 2023-09-22 at 09 48 04](https://github.com/facebookexperimental/Robyn/assets/42702165/4397a7fe-6c54-4dfb-a132-ad8c01805ed7)

And here is the same model when brought into R via robyn_write():

![Screenshot 2023-09-22 at 09 48 28](https://github.com/facebookexperimental/Robyn/assets/42702165/0443ad2e-6e60-4079-83fe-400714fbf411)

Thank you.

**Comment by gufengzhou** [comment]
Yes, because when `ts_validation = TRUE`, the objective function to be optimised is `nrmse_val`. When when` ts_validation = FALSE`, it's `nrmse_test`

**Comment by SeanRichterWalsh** [comment]
@gufengzhou  thanks for replying. I am still a bit confused though because when `ts_validation = FALSE `there is no test set, right? Is it that NRME train is optimised for? Thanks.

**Comment by gufengzhou** [comment]
Right sorry that was a typo. Just corrected it. It's train

**Comment by SeanRichterWalsh** [comment]
Clear. Thanks a lot.
